src/test/java/uk/gov/companieshouse/itemhandler/kafka/OrdersKafkaConsumerTest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.companieshouse.itemhandler.kafka;
+
+import org.junit.Assert;
+import org.junit.Test;
```

Comment:
Is there a specific reason why this is using JUnit 4?

Reply:
These tests have been updated to use JUnit5.